community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/NativeSchemaIndex.java

```diff
@@ -70,14 +70,16 @@ void instantiateTree( RecoveryCleanupWorkCollector recoveryCleanupWorkCollector,
         tree = new GBPTree<>( pageCache, storeFile, layout, 0, monitor, NO_HEADER_READER, headerWriter, recoveryCleanupWorkCollector );
     }
 
+    abstract String partName();
+
     private GBPTree.Monitor treeMonitor( )
     {
         return new GBPTree.Monitor.Adaptor()
         {
             @Override
             public void cleanupFinished( long numberOfPagesVisited, long numberOfCleanedCrashPointers, long durationMillis )
             {
-                monitor.recoveryCompleted( indexId, descriptor, map(
+                monitor.recoveryCompleted( indexId, descriptor, partName(), map(
```

Comment:
`partName()` seems so "fusion-aware" to me. And why not pass the absolute file name instead? The file would convey more information and you wouldn't need to add that partName() method at all, which seems to be implemented by populators too, unexpectedly.

Reply:
That's why I wrote `absolute` file name. That would work, right? It would give the whole information about index id, provider and the whole shebang, wouldn't it?

Reply:
You _did_ write that. I didn't read that however 😅  Yeah, that works! It will not give any information about label and property id, but the rest will be there.

Reply:
You are so very right, it won't. Yup it may be good enough to not be super unhelpful, at least!